data/json/furniture.json

```
@@ -866,7 +866,7 @@
     "type": "furniture",
     "id": "f_ball_mach",
     "name": "ball machine",
-    "description": "Remember when baseball was a thing?",
+    "description": "Remember when Skee-ball was a thing?  Well, the ball return doesn't seem to work without power and somebody's kicked in the coin slot.",
```

I'm not actually sure what type of machine this is truly meant to represent, so you may want to look at places where it is referenced more in-depth to get a better idea of the description (something I had to do for my descriptions).

It could be a tennis ball machine, a ping pong ball machine, a skee ball machine or baseball pitching machine. If you can reword it to be a bit more generic:

```suggestion
    "description": "An unpowered machine that seems like it could've been used to launch various balls for different types of sports.  It's only good for parts now if disassembled.",
```

---

> 
> 
> I'm not actually sure what type of machine this is truly meant to represent, so you may want to look at places where it is referenced more in-depth to get a better idea of the description (something I had to do for my descriptions).
> 
> It could be a tennis ball machine, a ping pong ball machine, a skee ball machine or baseball pitching machine. If you can reword it to be a bit more generic:

We talked about it on Discord and decided Skee-ball specifically. I can make it more generic though, if you like. And definitely add the disassembly bit.